geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/function/FastPropertyTest.scala

```diff
@@ -49,7 +49,7 @@ class FastPropertyTest extends Specification {
       val sf = new ScalaSimpleFeature("id", sft)
       sf.setAttributes(Array[AnyRef]("myname", "POINT(45 45)"))
 
-      val filter = ECQL.toFilter("testns:name = 'myname'", FastFilterFactory.factory)
+      val filter = FastFilterFactory.toFilter(sft, "testns:name = 'myname'")
```

Comment:
We implemented the namespace logic (which moved around in this PR) to handle queries in GeoServer where namespaced attributes were being used.  I suppose I'm wondering if that pathway will still work...  (Or if the move shifted the code out of the intended pathway.)